src/utilities/identities/identities.ts

```diff
@@ -58,7 +58,7 @@ export function useIdentities(): IdentitiesContextType {
   return useContext(IdentitiesContext);
 }
 
-async function getCurrentIdentity(): Promise<string | null> {
+export async function getCurrentIdentity(): Promise<string | null> {
```

Comment:
Is it imported anywhere?

Reply:
Good catch! Not anymore, I only needed that for an interim solution.